interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,100 @@ components:
       required:
         - uomDimensions
         - verticalReferenceType
-        - verticalReference
-        - longitude
-        - latitude
       properties:
         uomDimensions:
           $ref: '#/components/schemas/UomDimensions'
         verticalReferenceType:
           $ref: '#/components/schemas/VerticalReferenceType'
         verticalReference:
+          description: >-
+            Height above vertical reference datum, in units of uomDimensions.
           type: number
+          default: 0
         longitude:
+          description: >-
+            Longitude, degrees east of prime meridian.
           type: number
+          default: 0
         latitude:
+          description: >-
+            Latitude, degrees north of the equator.
           type: number
+          default: 0
 
-    UASZonesCheckRequest:
+    GeozonesCheckRequest:
       type: object
       required:
-        - position
-        - uSpaceClass
-        - acceptableRestrictions
-        - startDateTime
-        - endDateTime
+        - checks
       properties:
-        position:
-          $ref: '#/components/schemas/Position'
-        uSpaceClass:
-          $ref: '#/components/schemas/USpaceClass'
-        acceptableRestrictions:
+        checks:
           type: array
           items:
-            $ref: '#/components/schemas/Restriction'
-        startDateTime:
+            $ref: '#/components/schemas/GeozonesCheck'
+
+    GeozonesCheck:
+      type: object
+      required:
+        - filterSets
+      properties:
+        filterSets:
+          description: >-
+            Select Geozones which match any of the specified filter sets.
+          type: array
+          items:
+            $ref: '#/components/schemas/GeozonesFilterSet'
+
+    GeozonesFilterSet:
+      description: >-
+        Set of filters to select only a subset of Geozones.  Only Geozones which are applicable to all specified filters within this filter set should be selected.
+      type: object
+      properties:
+        position:
+          description: >-
+            If specified, only select Geozones encompassing this position.
+          anyOf:
+            - $ref: '#/components/schemas/Position'
+        after:
+          description: >-
+            If specified, only select Geozones which are defined at or after this time.
           type: string
           format: date-time
-        endDateTime:
+        before:
+          description: >-
+            If specified, only select Geozones which are defined at or before this time.
           type: string
           format: date-time
+        ed269:
```

Comment:
By grouping filter criteria by requirement source, we can add filter criteria for other contexts alongside existing filter criteria without affecting existing filter criteria.  When the replacement for ED-269 is release, we can leave this field and the tests built for it while still stepping up to and supporting the new ED-xxx by simply adding a new filter criteria group like `edXXX`.

Reply:
This is a great improvement to the initial design. Thank you!